packages/qocamlbrowser/qocamlbrowser.0.2.12/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "kakadu@pm.me"
+homepage: "https://github.com/Kakadu/QOcamlBrowser_quick"
+bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
+dev-repo: "git://github.com/Kakadu/QOcamlBrowser_quick"
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
+authors: "Kakadu"
+synopsis: "OCamlBrowser clone written with OCaml and QtQuick 2"
+
+build: [
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
+    {os != "macos"}
+  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]
+    {os != "macos"}
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH ./configure"] {os = "macos"}
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:$PATH make"] {os = "macos"}
+]
```

Comment:
I think it would be simpler and more maintainable to detect where the binaries you need are, in `conf-qt` and to use the configuration variable directly. (see the `conf-cmake` as a simple example and `conf-llvm` as a more developed one)

Reply:
I did it.

Reply:
OKay, I should probably do the same here too.

Reply:
Done!